packages/react-components/react-tabster/src/hooks/useModalAttributes.ts

```diff
@@ -5,9 +5,11 @@ import { useTabster } from './useTabster';
 
 export interface UseModalAttributesOptions {
   /**
-   * Traps focus inside the elements the attributes are applied
+   * Traps focus inside the elements the attributes are applied.
+   * It is recommended not to use the `absolute` variant since
+   * it forbigs users to tab out of the focus trap into the actual browser.
    */
-  trapFocus?: boolean;
+  trapFocus?: boolean | 'absolute';
```

Comment:
using a union type to avoid breaking API

Reply:
In the PR description you mention this is added for backward compatibility. Do you see this feature being removed in the future (i.e., in v10 or 11)?

Reply:
It's honestly hard to say. From an a11y PoV it's better in a focus trap to be able to tab into the parent window. Essentially a modal dialog just becomes the `document`.

However decades of focus trap being focus 'jail' have made most devs (even the a11y minded ones) kind of suspicious about something new.

You can see my branch is called `legacy-focus-trap` and honestly I would like to consider it legacy but I think mindset is not there yet.

Reply:
What do you think about `legacy` instead of `absolute`?

Reply:
can we introduce this legacy behaviour as separate prop ? this sounds like API code smell - have union of boolean and string literal ( there was extended convo about this for react-card focus PR where we agreed on certain patterns to follow )

Reply:
I added it as a separate `legacyTrapFocus` prop per popular suggestion